src/argo/workflow.yaml

```diff
@@ -54,20 +54,34 @@ spec:
                   value: extracted-clinvar-json
                 - name: rows-per-output
                   value: 1024
+        - - name: upload-raw-archive
```

Comment:
As written, this will run every step sequentially. Nothing in here takes long enough to really justify us worrying about parallelism, _but_ if we wanted to get practice with it we could try switching to a DAG mode:
```
       Download XML
           /  \
Upload XML      XML->JSON
                   |
             Upload JSON
```
Thoughts?

Reply:
Implementing it this way, I agree it is the "right" way and good practice.